include/libnuraft/launcher.hxx

```diff
@@ -45,7 +47,9 @@ public:
                           ptr<logger> lg,
                           int port_number,
                           const asio_service::options& asio_options,
-                          const raft_params& params);
+                          const raft_params& params,
+                          const raft_server::init_options opt = raft_server::init_options(),
```

Comment:
Could you please use reference?
```C++
const raft_server::init_options& opt = raft_server::init_options()
```

Reply:
Agree.

Actually, I was just copied from your unit test case :p
https://github.com/eBay/NuRaft/blob/75017885cd0ad5f10c805f035fa5b44ae991aa3a/tests/unit/raft_package_fake.hxx#L51-L54

Reply:
Oh, good catch, it also needs to be fixed. Thanks :)